packages/datadog-plugin-express/src/code_origin.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+const { entryTags } = require('../../datadog-code-origin')
+const Plugin = require('../../dd-trace/src/plugins/plugin')
+const web = require('../../dd-trace/src/plugins/util/web')
+
+class ExpressCodeOriginForSpansPlugin extends Plugin {
```

Comment:
If I'm understanding our wiki page right we'll ultimately need one of these code origin classes / files for all web servers and for all outbound / client integrations?